### ISSUE TITLE
series/seriesOf should be partOf/hasPart

### ISSUE THREAD

**Issue by nichtich** [comicmeta/ComicBookOntology#18]
the current mapping is reverse

**Comment by seanpetiya** [comment]
Can you clarify? I think the current mapping is correct.

cbo:Comic cbo:series (hasPart) cbo:Series
cbo:Series cbo:seriesOf (partOf) cbo:Publication

**Comment by seanpetiya** [comment]
Could you clarify? I think the current mapping (although imperfect) might be correct.

cbo:Comic cbo:series (hasPart) cbo:Series
cbo:Series cbo:seriesOf (partOf) cbo:Publication

**Comment by nichtich** [comment]
I think this origins from the different mindset of series-oriented comics in the US and book oriented comics in Europe: does a comic consist of multiple series' or does a series contain multiple comics? Is one-shot a special case of a series or does a series consist of multiple parts? I think both should be possible. 

The current hasPart properties for comparision:

* Comic -hasPart(series) -> Series (comic can contain series)
* Series -hasPart(volume)-> Volume (series can contain volumes)
* Volume -hasPart(issue) -> Issue (volume can contain issues)
* Issue -hasPart(collects) -> Issue (issues can contain other issues)
* Issue -hasPart(story)-> Story (issue can contains stories)
* Strory -hasPart(storyArc) -> Story (story can contain stories)
* Collection -hasPart(box) -> Box
* Collection -hasPart(item) -> Item

[Here an example](https://www.amazon.de/F-Z-Klassiker-Comic-Literatur/lm/1O47HSLZTBVVT) of a series that contains multiple comics.

The underlying question is what a "Comic" actually refers to in CBO. In my point of view it is a rather broad concept: some series are comics, some volumes are comics and some stories are comics but some comics consist of multiple series/volumes/issues.

**Comment by seanpetiya** [comment]
Yes, the "Comic" class in CBO is intended to be as broad as possible to incorporate all conceptualizations of a "Comic" work, i.e. story, panel, including the entire body of work that represents a comic, including one or more comic series/volumes. Each of which may have one or more issues. So, technically, both of your examples should be possible.

For example, the parts of the comic "X-Men" in the broadest sense would include all related works and all manifestations of those works (series): the "Uncanny X-Men" comic-book series, but also the "Uncanny X-men" omnibus series, etc. With the option to divide series up by volume, for example the [Uncanny X-Men omnibus](http://www.worldcat.org/search?q=ti%3Auncanny+x-men+omnibus+volume&qt=notfound_page&search=Search) contains three "volumes". 

The intention of the model is to accommodate all perspectives; comics as series, comics as books, etc. The actual domains of the hasPart/isPartOf properties in CBO, however, can definitely use some tweaking... maybe starting with the issue property? The domain should probably be Volume or Series...
